core/server/src/main/java/alluxio/proxy/AlluxioProxy.java

```diff
@@ -47,9 +47,12 @@ public static void main(String[] args) {
       System.exit(-1);
     }
 
-    if (!Configuration.containsKey(PropertyKey.MASTER_HOSTNAME)) {
-      System.out.println("Cannot start proxy; master hostname is not configured. Please set "
-          + PropertyKey.MASTER_HOSTNAME.toString() + " in alluxio-site.properties.");
+    if (!Configuration.masterHostConfigured()) {
+      System.out.println(String.format(
+          "Cannot start proxy; master hostname is not configured. "
+              + "Please either configure %s in alluxio-site.properties, or configure the proxy to "
```

Comment:
`Configuration.SITE_PROPERTIES` instead of `alluxio-site.properties`? also for the other two messages.